include/vara/Feature/Constraint.h

```diff
@@ -25,55 +25,166 @@ class Constraint {
   ConstraintKind Kind;
 };
 
-class ConstraintOperator {
-public:
-  /// TODO: Numeric Operators are missing (e.g. GREATER, LESS, MINUS, ADD, MULTIPLY, DIVISION)
-  enum class OperatorKind { OK_NOT, OK_NEGATION, OK_AND, OK_OR, OK_XOR, OK_EQUALS, OK_IMPLIES, OK_EQUIVALENCE };
-  ConstraintOperator(OperatorKind Operator) : Operator(Operator) {};
-
-  [[nodiscard]] std::string toString() {
-    switch(Operator) {
-    case OperatorKind::OK_NOT: return "!";
-    case OperatorKind::OK_NEGATION: return "~";
-    case OperatorKind::OK_AND: return "&";
-    case OperatorKind::OK_OR: return "v";
-    case OperatorKind::OK_EQUALS: return "=";
-    case OperatorKind::OK_IMPLIES: return "=>";
-    case OperatorKind::OK_EQUIVALENCE: return "<=>";
-    case OperatorKind::OK_XOR: return "^";
-    default: return "";
-    }
-  };
+class BoolschesConstraint {
```

Comment:
```suggestion
class BooleanConstraint {
```
This should be spell in English I guess :)